docs/tutorials.rst

```diff
@@ -0,0 +1,180 @@
+==============================================================================
+A user guide for tool chain of automated tutorials for the MSUI user interface
+==============================================================================
+
+The tool chain for tutorials are automatically generated tutorials by use of python scripts to automate the software to
+perform different tasks and hence, demonstrating and explaining to the user various parts of the MSS software. It tells
+the user about the different technical aspects of MSS which a new user would take time to know that, otherwise.
+
+There are nine such total automating scripts that are meant for explaining nine such parts or sub parts of MSS software
+and it's use.
+Listing them below as:
+
+* tutorial_hexagoncontrol.py for explaining hexagon control in table view.
+
+* tutorial_kml.py for explaining kml dockwidget in topview.
+
+* tutorial_mscolab.py for understanding mscolab feature of MSS.
+
+* tutorial_performancesettings.py for giving a demo what is there in performance settings of table view.
+
+* tutorial_remotesensing.py for demonstrating remote sensing dockwidget of topview.
+
+* tutorial_satellitetrack.py for demonstrating satellite track dockwidget of topview.
+
+* tutorial_views.py for demonstrating the top view, sideview, table view and linear view together.
+
+* tutorial_waypoints.py for demonstrating how to place waypoints in topview.
+
+* tutorial_wms.py for explaining web map service section of topview.
+
+Other important files with their functions are as follows:
+
+* screenrecorder.py for opening a screen recorder window which is a small window in the top right corner of the
+  rectangular recording area on the screen. The window gets closed when you press 'q' keeping the window in focus. This
+  will also stop the recording of the screen when window gets closed.
+
+* cursor.py which is used for generating the image of the present cursor image.
+
+* audio.py which translates and converts into speech the texts and saves them as .mp3 files.
+
+Getting Started..
+=================
+
+On the Anaconda terminal, type the following ::
+
+ cd ..../MSS/$
+ $ conda activate mssdev
+
+
+ (mssdev)$ conda install --file requirements.d/tutorials.txt
+ (mssdev)$ conda deactivate
+ (base)$ conda activate mssdev
+
+
+This will install all the dependencies required for running of the tutorials.
+
+
+**On Linux additionally** ::
+
+    $ sudo apt-get install scrot
+    $ sudo apt-get install python3-tk
+    $ sudo apt-get install python3-dev
+    $ sudo apt-get install libx11-dev libxext-dev libxfixes-dev libxi-dev
+
+
+Now, just go into the _../MSS/tutorials/_ directory ::
+
+    $ cd ../MSS/tutorials/
+
+
+Keep the following things in mind before running a script
+
+* You should have only an **US keyboard layout**. If you have a different keyboard layout, you just need to change it to
```

Comment:
this should be on a more prominent place. e.g. before Getting Started

System requirements:
- full hd resolution 
- us keyboard configured

I it is also required to have a full HD resolution, not more, not less

Reply:
Okay, getting these things corrected.